docker-compose.yml

```diff
@@ -41,6 +41,7 @@ services:
       - PFS_KEYSTORE_FILE=/keystore/${KEYSTORE_FILE}
       - PFS_PASSWORD=${PASSWORD}
       - PFS_ETH_RPC=${ETH_RPC}
+      - PFS_ACCEPT_DISCLAIMER=${ACCEPT_PFS_DISCLAIMER}
```

Comment:
Is this necessary with the command above?

Reply:
maybe it's nice to remove it from the command, and only use it here. And then define it as `ACCEPT_PFS_DISCLAIMER=TRUE`

Reply:
yes I think it is, since this command will run in a different environment